### DISCUSSION TITLE
Streams aren't loading

### DISCUSSION THREAD

**Discussion by neurodiverseEsoteric** [streamlink/streamlink-twitch-gui#776] [General]
The only output from Konsole is:
```
Error: Process exited with code 1
[error][StreamingService]
```
Sorry, I got a 404 message when I hit the submit button, so I thought it didn't actually record the bug...

**Comment by bastimeyer** [comment]
No need to post this here twice.
https://github.com/streamlink/streamlink-twitch-gui/issues/774
